### PR TITLE
fix(settings): Fix settings name

### PR DESCRIPTION
**Pull request by mickenordin**
I did a bunch more testing and in #901 it seems I got the name of the config key wrong in some places.

This patch fixes the issue.

### CODE REVIEW

File: lib/SAMLSettings.php [hunk 1]
```diff
@@ -167,6 +166,7 @@ public function getOneLoginSettingsArray(int $idp): array {
 					'url' => $this->configurations[$idp]['idp-singleSignOnService.url'] ?? '',
 				],
 				'x509cert' => $this->configurations[$idp]['idp-x509cert'] ?? '',
+				'passthroughParameters' => $this->configurations[$idp]['idp-passthroughParameters'] ?? '',
```
Comment (blizzz) [inline lib/SAMLSettings.php:169]: is this now correct with this 'idp-' text prefix?

**Review by blizzz — COMMENTED** [review]
one question

File: lib/SAMLSettings.php (hunk 1 above)
Comment (mickenordin) [reply lib/SAMLSettings.php:169]: I now have this code running in 8 differnt sites, all working, so I think we are good.

**Review by blizzz — APPROVED** [review]

File: lib/SAMLSettings.php (hunk 1 above)
Comment (blizzz) [reply lib/SAMLSettings.php:169]: Yes, all good, boils down to the fact that we have to convert it to a php-saml compatible array structure. Hence the toying with the prefix.

**Pull request merged**